hazelcast/src/main/java/com/hazelcast/console/ConsoleApp.java

```diff
@@ -99,12 +100,18 @@
     private boolean silent;
     private boolean echo;
 
+    private final PrintStream systemOut;
```

Comment:
It seems that `master` already has a similar field:
https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/console/ConsoleApp.java#L104

Could you merge with this change from `master`? This will simplify forward-port and make branches more consistent.

Reply:
addressed in https://github.com/hazelcast/hazelcast/pull/17008/commits/490da7c2ae9bafb6f9099a7057e85c5dd539e59a